File: 8KYU/Sum of Positive/solution.cpp
#include <vector>

int positive_sum (const std::vector<int> arr){
  int total = 0;
  
  for(int i = 0; i < arr.size(); i++) {
    if(arr[i] > 0) {
      total += arr[i];
    }
  }
  
  return total;
}

File: 8KYU/Sum of Positive/solution.php
function positive_sum($arr) {
  $total = 0;
  
  for($i = 0; $i < count($arr); $i++) {
    if($arr[$i] > 0) {
      $total = $total + $arr[$i];
    }
  }
    
  return $total;
}